src/routes/game/components/dialogs/components/gameOver/GameOverDialog.vue

```diff
@@ -1,66 +1,47 @@
 <template>
   <BaseDialog id="game-over-dialog" v-model="show">
     <template #title>
-      <h1 :data-cy="headingDataAttr" :class="isMobilePortrait ? 'text-h4' : ''">
+      <h1 :data-cy="headingDataAttr" :class="isMobilePortrait ? 'text-h4' : 'heading'">
         {{ heading }}
       </h1>
-      <v-img
-        v-if="currentMatch && !isMobilePortrait"
-        :src="logoSrc"
-        :data-cy="logoDataAttr"
-        class="logo-image-match"
-      />
     </template>
 
     <template #body>
-      <template v-if="isMobilePortrait || !currentMatch">
-        <div class="d-flex justify-center">
-          <v-img
-            :src="logoSrc"
-            :data-cy="logoDataAttr"
-            :class="isMobilePortrait ? 'small-logo-image' : 'logo-image'"
+      <MatchScoreCounter
+        :wins="wins"
+        :losses="losses"
+        :stalemates="stalemates"
+        :latest-result="latestResult"
+      />
+      <section class="d-flex justify-space-around mt-6 mb-8">
+        <template v-if="matchIsOver">
+          <MatchWonOrLostIndicator
+            :username="gameStore.player.username"
+            :won-match="playerWinsMatch"
+            data-cy="player-match-result"
+          />
+          <MatchWonOrLostIndicator
+            :username="gameStore.opponent.username"
+            :won-match="opponentWinsMatch"
+            data-cy="opponent-match-result"
+          />
+        </template>
+        <template v-else>
+          <LobbyPlayerIndicator
+            :player-username="gameStore.player.username"
+            :player-ready="gameStore.iWantRematch"
+            small
+            data-cy="my-rematch-indicator"
```

Comment:
Todo: rename and relocate this component to be generic, now that it's used outside the lobby